### ISSUE TITLE
[FEATURE-REQUEST] Alpha release publish on conda-forge

### ISSUE THREAD

**Issue by alippai** [vaexio/vaex#1103]
I see that you publish the alpha releases on pip. Could you add conda-forge deployments too?

**Comment by maartenbreddels** [comment]
Hi Adam,

To conda-forge will be difficult to automate I think,  we could consider a similar approach to Arrow  https://arrow.apache.org/docs/python/install.html#installing-nightly-packages

would that work for you?

cheers,

Maarten

**Comment by alippai** [comment]
Yes, that's a totally awesome solution. Also treat this as low prio request, please. I know you are a small team, focus on the features you think are the most important for the next stable release :)
